assets/dir-index-html/dir-index.html

```diff
@@ -53,10 +53,13 @@
           {{ .Hash }}
         </div>
         {{ end }}
+        <p style="margin-bottom: 0">
+          <a href="?format=json">Preview as JSON.</a> Download as: <a href="?format=car">CAR</a>, <a href="?format=tar">TAR</a>, <a href="?format=dag-json">DAG-JSON</a>, <a href="?format=dag-cbor">DAG-CBOR</a>.
```

Comment:
Mind adding `rel="nofollow, noindex"`  to all `?format=` everywhere? We don't want bots to introduce unnecessary burden. (https://en.wikipedia.org/wiki/Noindex)